src/components/WorkflowForms/NewClientSignup/GatherInfo/hooks/useProviders.js

```
@@ -0,0 +1,269 @@
+            // Map the data filtered by the backend directly
+            setAvailableProviders(result.data.map(provider => ({
+               id: provider.xref,
+               xref: provider.xref,
+               name: provider.name,
+               type: provider.types ? provider.types[0] : '',
+               phone: provider.phone,
+               address: provider.address,
+               website: provider.website
+            })));
+         } else {
+         }
```

ESLint

[error] Empty block statement.

(no-empty)